packages/desktop-client/src/components/modals/GoalTemplateModal.tsx

```diff
@@ -0,0 +1,170 @@
+import { Heading } from 'react-aria-components';
+import { useTranslation } from 'react-i18next';
+
+import { theme } from '../../style';
+import { Link } from '../common/Link';
+import { Modal, ModalCloseButton, ModalHeader } from '../common/Modal';
+import { Text } from '../common/Text';
+import { View } from '../common/View';
+import { TableHeader, Row, Field } from '../table';
+
+export function GoalTemplateModal() {
+  const { t } = useTranslation();
+  return (
+    <Modal name="goal-templates" containerProps={{ style: { width: 1000 } }}>
+      {({ state: { close } }) => (
+        <>
+          <ModalHeader
+            title={t('Goal Templates')}
+            rightContent={<ModalCloseButton onPress={close} />}
+          />
+          <Heading level={3}>{t('Weekly')}</Heading>
+          <View>
+            <TableHeader
```

Comment:
Try this instead:

```
      <TableHeader>
        <Field width="flex">
          <Trans>Syntax</Trans>
        </Field>
        <Field width="flex">
          <Trans>Description</Trans>
        </Field>
      </TableHeader>
```

Reply:
Thanks, that seems to work. Odd it works fine in other files. Will be pushing the change shortly